website/admin/views/__init__.py

```diff
@@ -44,3 +37,33 @@ def logout():
     utilities.session.logout()
     flash('Successfully logged out', 'success')
     return redirect(url_for('public.index'))
+
+
+@admin_site.route('/triggers')
```

Comment:
You probably need the user that is being seen by the admin as a parameter. Or how would you handle this?